docs/macOS-install-and-tutorial-instructions-for-conda.md

```diff
@@ -0,0 +1,35 @@
+# Installing brainlit on macOS using conda environment:
+1. Installation of conda environment
+    * [get Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html)
+    * create a virtual environment: `conda create --name brainlit python=3.8`
+    * activate the environment: `conda activate brainlit`
+2. Install brainlit
+    * Option 1: install from pypi
+        * install brainlit: `pip install brainlit`
+    * Option 2: install brainlit from source
+        * cd to directory you want to install brainlit in
+        * clone the repo: git clone https://github.com/neurodata/brainlit.git
+        * cd into the repo: `cd brainlit`
+        * install brainlit: `pip install -e .`
+
+# Downloading brains tutorial on iOS using conda environment, using jupyter notebook: 
```

Comment:
macOS not iOS

Reply:
Changed PR name to macOS, and also renamed to `macOS-install-and-tutorial-instructions-conda.md`/changed iOS to macOS in all sections in newest commit